IntroductionAppCalculateUserAge/Program.cs

```diff
@@ -6,7 +6,16 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.Write("Enter year of your birth:");
+            var yearOfBirch = Convert.ToInt32(Console.ReadLine());
+            var currentYear = DateTime.Today.Year;
+            var userAge = currentYear - yearOfBirch;
+            var adultMarker = 18;
```

Comment:
Lets make this variable constant. According to conventions move it to the start of your methods and add 'const' word.